client-tmp/angular/custom-webpack.config.js

```diff
@@ -0,0 +1,53 @@
+const path = require('path');
+const { resolveDefineConfig } = require('@genesislcap/build-kit');
+const { config } = require('dotenv');
+const { DefinePlugin } = require('webpack');
+const pkg = require('./package.json');
+const pkgConfig = Object.fromEntries(
+  Object.entries(pkg.config).map(([key, value]) => [
+    key,
+    JSON.stringify(value),
+  ]),
+);
+config();
+
+module.exports = {
+  plugins: [
+    new DefinePlugin(
+      resolveDefineConfig([
+        'API_HOST',
```

Comment:
API_HOST should only be passed in dev mode (normally you can check for that using process.env.NODE_ENV)

Reply:
I tested it again and foundation-login fetches this API_HOST from package.json -> config - so I'm removing it here for now